filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/01TrustedNetwork

```diff
@@ -0,0 +1,16 @@
+{
+    use esmith::NetworksDB;
+    my $ndb = esmith::NetworksDB->open_ro();
+
+    #return a cidr for localnetwork
+    foreach (map {
+    my ($addr, $mask) = split('/', $_);
+        esmith::util::computeLocalNetworkShortSpec($addr, $mask || '255.255.255.255');
+            } $ndb->local_access_spec() ) {
```

Comment:
You could find a compact API for trusted networks in NethServer::TrustedNetworks Perl module, which is the base for local_access_spec()

Some examples

https://github.com/search?q=org%3ANethServer+NethServer%3A%3ATrustedNetworks&type=Code

Reply:
ok